nixos/modules/programs/environment.nix

```diff
@@ -38,9 +38,6 @@ in
         QT_PLUGIN_PATH = [ "/lib/qt4/plugins" "/lib/kde4/plugins" ];
         QTWEBKIT_PLUGIN_PATH = [ "/lib/mozilla/plugins/" ];
         GTK_PATH = [ "/lib/gtk-2.0" "/lib/gtk-3.0" ];
-        XDG_CONFIG_DIRS = [ "/etc/xdg" ];
```

Comment:
Where did this go?

Reply:
Whoops, left over from when I did try having a base-directories module.